src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -114,7 +118,12 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new GitHubPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
+                    ParameterizedJobMixIn scheduler = new ParameterizedJobMixIn() {
```

Comment:
I don't think it's a ```scheduler```, I know what it is. I just said I'm using it to schedule a job.
I repeat, tell me your name proposal and I'll change it.